src/utils/styleParser.js

```diff
@@ -11,6 +11,9 @@ module.exports.parse = function (value) {
   var parsedData;
   if (typeof value !== 'string') { return value; }
   parsedData = styleParser.parse(value);
+  // The style parser returns an object { "" : "test"} when
```

Comment:
This could fit in one line. `Handle style parser returning {"": <string>} when fed a string.`